packages/pluggableWidgets/video-player-web/package.json

```diff
@@ -24,7 +28,8 @@
     "pretest:e2e": "node ../../../scripts/test/updateAtlas.js",
     "test:e2e": "pluggable-widgets-tools test:e2e:web --mx-version 8",
     "test:e2e:dev": "pluggable-widgets-tools test:e2e:web:dev",
-    "release": "pluggable-widgets-tools release:web"
+    "release": "pluggable-widgets-tools release:web",
+    "releaseMarketplace": "node ../../../scripts/release/marketplaceRelease.js"
```

Comment:
To keep script names inline with NPM convention I would opt for: `release:marketplace`

Reply:
Fixed